app/jobs/preassembly_job.rb

```diff
@@ -8,8 +8,6 @@ def perform(job_run)
     job_run.started
     bc = job_run.batch_context
     result = bc.batch.run_pre_assembly
-    job_run.output_location = bc.progress_log_file
-    job_run.save!
     result ? job_run.completed : job_run.errored
```

Comment:
the "output_location" in the database used to track state (it's existence holding the name of the progress log file indicated job completion)...we now track state directly so copying this filename is not necessary 

instead, this field is now only used to store the discovery report job (which it also did before) and we get the progress log file location from the batch_context when we want to download it